lib/middleware/metrics.rb

```diff
@@ -28,6 +28,18 @@ def is_private_ip?(env)
       !!(ip && ip.to_s =~ PRIVATE_IP)
     end
 
+    def is_trusted_ip?(env)
+      if
```

Comment:
This looks weird. It's not immediately clear that it's returning `nil`... Maybe something like this?

```ruby
def is_trusted_ip?(env)
  return false if GlobalSetting.prometheus_trusted_ip_whitelist_regex.empty?
  # ...
end
```

Reply:
Yeah. Because I learned Lisp forty years ago I don't know that nil and false are different.

I'll fix it in the next day or so. 

Thanks

Reply:
Yeah. Because I learned Lisp forty years ago I don't know that nil and false are different.

I'll fix it in the next day or so. 

Thanks